src/elements/Button.tsx

```diff
@@ -0,0 +1,120 @@
+import { FunctionComponent } from "react";
+import {
+  CircularProgress,
+  Button as MUIButton,
+  ButtonProps as MUIButtonProps,
+} from "@mui/material";
+import theme from "theme";
+
+interface ButtonProps extends Omit<MUIButtonProps, "color" | "variant"> {
+  readonly color?:
```

Comment:
I thought about doing that, the reason i didnt was because the mocks didnt have a "newm" button variant but had a "white" (which is not in the gradients). Currently, every gradient color specified here has a solid color equivalent , with the exception of "white" but added some logic to point to the music gradient. Going to leave it as is for now and we can update it later if we need to